modules/machine-health-checks-pausing.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+
+// * updating/updating-cluster-cli.adoc
+
+[id="machine-health-checks-pausing_{context}"]
+= Pausing a MachineHealthCheck resource
+
+During the upgrade process, nodes in the cluster might become temporarily unavailable. The `MachineHealthCheck` might identify such nodes as unhealthy and reboot them. To avoid rebooting such nodes, pause all the `MachineHealthCheck` resources before updating the cluster.
+
+.Prerequisites
+
+* Install the `oc` command line interface.
```

Comment:
There's a standard way to say this: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/term_glossary.adoc#openshift-cli

```suggestion
* Install the OpenShift CLI (`oc`).
```